example/demo/src/Page/BlocksPage.php

```diff
@@ -80,10 +78,7 @@ public function lorem(): ParagraphWidget
         $text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.';
 
         return ParagraphWidget::fromText(
-            Text::styled(
-                $text,
-                Style::default()->fg(AnsiColor::DarkGray)
-            )
+            Text::parse("<fg=darkgray>{$text}</>")
```

Comment:
minor: use `sprintf` instead

Reply:
Yes, it probably exists. I'll see this later.